components/hero/hero.tsx

```diff
@@ -0,0 +1,46 @@
+import Media from '../media';
+import hero from '../../public/images/hero.webp';
+
+import style from './hero.module.scss';
```

Comment:
Yeah good question. It's because we're using CSS modules, which Next includes as an out-of-the-box way to [style components](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css). This method of importing and then referencing the style object from the file is how you apple the class names and styles.

Reply:
I see, thanks for clarifying!